notebooks/en/enterprise_cookbook_gradio.ipynb

```diff
@@ -0,0 +1,5796 @@
+{
```

Comment:
<p>For Tips, you can use these nice callouts:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">&gt; [!TIP]
&gt; The output from the call above is a made up meeting...
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>